components/pagination/Pagination.tsx

```diff
@@ -50,6 +51,10 @@ export default class Pagination extends React.Component<PaginationProps, {}> {
   }
 
   render() {
+    // When autoHide is true and there is only 1 page, hide the pager
+    if (this.props.autoHide === true && (this.props.total || 0) <= (this.props.pageSize || 10)) {
```

Comment:
The `defaultPageSize` comes from `rc-pagination`, I think `rc-paginaion` is a better place a add this feature?

Reply:
Sounds good, done